swift/ql/lib/codeql/swift/security/CleartextStorageDatabaseExtensions.qll

```diff
@@ -15,16 +15,16 @@ import codeql.swift.dataflow.ExternalFlow
 abstract class CleartextStorageDatabaseSink extends DataFlow::Node { }
 
 /**
- * A sanitizer for cleartext database storage vulnerabilities.
+ * A barrier for cleartext database storage vulnerabilities.
  */
-abstract class CleartextStorageDatabaseSanitizer extends DataFlow::Node { }
+abstract class CleartextStorageDatabaseBarrier extends DataFlow::Node { }
```

Comment:
This is technically a breaking change (since queries can depend on the name of this abstract class), but since we don't do proper deprecation cycles for Swift yet I guess all is fine 🎉.

Reply:
Yes, this is why I wanted to push the change in before public beta - fewer users potentially affected.